### PR TITLE
[debian] increment debian compatibility to 10 to enable parallel package build

### PR DESCRIPTION
**Pull request by stepanblyschak**

### CODE REVIEW

**Comment by lguohan** [comment]
retest this please

**Comment by liat-grozovik** [comment]
retest this please

**Comment by kcudnik** [comment]
probably because of parallelism this will not pass unittests, since mlnx.pl and brcm.pl are not designed to work in parallel, they need to be executed separately, since each of those instances uses vssyncd process to perform tests, and when executed in parallel, it will conflict

**Comment by lguohan** [comment]
how can we solve this issue?

**Comment by kcudnik** [comment]
not easy, since single test requires a vssyncd to be present as a process, a clean redis db and so on,
we can run each test in separate docker XD, then it should work

File: configure.ac [hunk 1]
```diff
@@ -3,7 +3,7 @@ AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_SRCDIR([])
 AC_CONFIG_AUX_DIR(config)
 AM_CONFIG_HEADER(config.h)
-AM_INIT_AUTOMAKE([subdir-objects foreign])
+AM_INIT_AUTOMAKE([subdir-objects foreign serial-tests])
```
Comment (kcudnik) [inline configure.ac]: great

**Pull request merged**